test/unit/api_spec.js

```diff
@@ -38,12 +38,31 @@ describe('api', function() {
       return data !== undefined;
     }, 20000);
   }
+
   describe('PDFJS', function() {
     describe('getDocument', function() {
       it('creates pdf doc from URL', function() {
-        var promise = PDFJS.getDocument(basicApiUrl);
-        waitsForPromiseResolved(promise, function(data) {
-          expect(data instanceof PDFDocumentProxy).toEqual(true);
+        var getDocumentPromise = PDFJS.getDocument(basicApiUrl);
```

Comment:
(no action requred by worth mentioning: getDocument returns loadingTask, which contains the promise; we probably need to review the code base to refactor variable names)